evm/src/cpu/kernel/asm/core/call.asm

```diff
@@ -12,18 +12,21 @@ global sys_call:
     // stack: kexit_info, gas, address, value, args_offset, args_size, ret_offset, ret_size
     %create_context
     // stack: new_ctx, kexit_info, gas, address, value, args_offset, args_size, ret_offset, ret_size
+    // TODO: Consider call depth
 
     // Each line in the block below does not change the stack.
     DUP4 %set_new_ctx_addr
     %address %set_new_ctx_caller
     DUP5 %set_new_ctx_value
-    DUP5 DUP5 %address %transfer_eth
+    DUP5 DUP5 %address %transfer_eth %jumpi(panic) // TODO: Fix this panic.
```

Comment:
Ah thanks for fixing that. Looks like the same `%transfer_eth` bug occurs further down also.